numpy/core/tests/test_longdouble.py

```diff
@@ -97,6 +99,22 @@ def test_fromfile_bogus(self):
             res = np.fromfile(path, dtype=float, sep=" ")
         assert_equal(res, np.array([1., 2., 3.]))
 
+    def test_fromfile_stdout(self):
+        p = sp.Popen([sys.executable, '-c', 'print("numpy is not np")'],
+                     stdout=sp.PIPE)
+        arr = np.fromfile(p.stdout, dtype=np.uint8, count=len('numpy is not np\n'))
+        assert_array_equal(arr.tostring(),
+                           np.asarray('numpy is not np\n', dtype='|S'))
+
+    def test_fromfile_stdout_unbuffered(self):
+        p = sp.Popen([sys.executable, '-c', 'print("numpy is not np")'],
+                     stdout=sp.PIPE, bufsize=0)
+        # Don't test the line ending char since it is platform dependent.
```

Comment:
Could use `import sys; sys.stdout.write(...)` above to avoid needing to do this.

Reply:
stdout would write to something right? presumably the screen?

Reply:
sorry for using CIs so much, compiling this stuff on windows is a pain.o

Reply:
there would be a `os.mkfifo`, but only on unix...

Reply:
`stdout` writes to exactly the same thing as `print` - which as configured with your `Popen` call, is the pipe.

Reply:
CI isn't so bad these days with azure-pipeline speeding things up. It really makes a difference for development. Thanks @tylerjereddy .

Reply:
ohhhh, you mean in the subprocess. sorry eric.